knowledge-base/mislav/bump-homebrew-formula-action/action-security.yml

```diff
@@ -0,0 +1,9 @@
+name: bump-homebrew-formula
+github-token:
+  environment-variable-name: GITHUB_TOKEN
+  permissions:
+    contents: write
```

Comment:
@Devils-Knight I think github token is only used to download something, not to write. also I think it just depends on whether it is set, and not on `push-to`. 

> You should enable GITHUB_TOKEN only if the repository that runs this Action is private and if COMMITTER_TOKEN has the public_repo scope only. GITHUB_TOKEN will be used for verifying the SHA256 sum of the downloadable archive for this release.